api/nakadi-event-bus-api.yaml

```diff
@@ -1029,7 +1024,8 @@ definitions:
           If not explicitly set, default value will respect the definition of the `EventType.category`.
         type: array
         items:
-          $ref: '#/definitions/EventValidationStrategy'
+          type: string
+        default: ['schema-validation']
```

Comment:
I think we call it `event-body-must-respect-schema`. Should we change it here in the API or in our code?